src/test/java/org/apache/ibatis/executor/loader/ResultLoaderMapTest.java

```diff
@@ -0,0 +1,55 @@
+/**
+ *    Copyright 2009-2019 the original author or authors.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package org.apache.ibatis.executor.loader;
+
+import java.sql.SQLException;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class ResultLoaderMapTest {
+
+  @Test
+  void hasLoaderInputNotNullOutputFalse() {
+    final ResultLoaderMap objectUnderTest = new ResultLoaderMap();
```

Comment:
use the actual name of the object if you can, rather than `objectUnderTest`, i.e. `resultLoaderMap = new ResultLoaderMap()`. Same for the rest of the tests.